client/src/snippets.json

```
@@ -24,17 +38,114 @@
         "prefix": "gx-conditional-select",
         "body": [
             "<conditional name=\"$1\">",
-            "    <param name=\"$2\" type=\"select\" label=\"$3\">",
-            "        <option value=\"$4\">TODO: option $4</option>",
+            "    <param name=\"$2\" type=\"select\" label=\"$3\" help=\"$4\">",
             "        <option value=\"$5\">TODO: option $5</option>",
+            "        <option value=\"$6\">TODO: option $6</option>",
             "    </param>",
-            "    <when value=\"$4\">",
+            "    <when value=\"$5\">",
             "        $0",
             "    </when>",
-            "    <when value=\"$5\">",
+            "    <when value=\"$6\">",
             "    </when>",
             "</conditional>"
         ],
         "description": "Conditional parameter select with two options"
+    },
+    "Select parameter with 3 options": {
+        "prefix": "gx-select-3-param",
+        "body": [
+            "<param name=\"$1\" type=\"select\" label=\"$2\" help=\"$3\">",
+            "    <option value=\"$4\">TODO: option $4</option>",
+            "    <option value=\"$5\">TODO: option $5</option>",
+            "    <option value=\"$6\">TODO: option $6</option>",
+            "</param>"
+        ],
+        "description": "Select parameter with 3 options"
+    },
+    "Select parameter with 5 options": {
+        "prefix": "gx-select-5-param",
+        "body": [
+            "<param name=\"$1\" type=\"select\" label=\"$2\" help=\"$3\">",
+            "    <option value=\"$4\">TODO: option $4</option>",
+            "    <option value=\"$5\">TODO: option $5</option>",
+            "    <option value=\"$6\">TODO: option $6</option>",
+            "    <option value=\"$7\">TODO: option $7</option>",
+            "    <option value=\"$8\">TODO: option $8</option>",
+            "</param>"
+        ],
+        "description": "Select parameter with 5 options"
+    },
+    "Boolean parameter": {
+        "prefix": "gx-boolean-param",
+        "body": [
+            "<param argument=\"$1\" type=\"boolean\" truevalue=\"$2\" falsevalue=\"\" label=\"$3\" help=\"$4\" />"
```

Should we reuse here the $1 place holder to repeat the same value in `argument` and `truevalue`?
```suggestion
            "<param argument=\"$1\" type=\"boolean\" truevalue=\"$1\" falsevalue=\"$2\" label=\"$3\" help=\"$4\" />"
```

---

Nice catch!